ReactAndroid/build.gradle

```diff
@@ -242,16 +242,22 @@ task buildReactNdkLib(dependsOn: [prepareJSC, prepareBoost, prepareDoubleConvers
             "REACT_COMMON_DIR=$projectDir/../ReactCommon",
             "REACT_SRC_DIR=$projectDir/src/main/java/com/facebook/react",
             '-C', file('src/main/jni/react/jni').absolutePath,
-            '--jobs', project.hasProperty("jobs") ? project.property("jobs") : Runtime.runtime.availableProcessors()
+            '--jobs', project.findProperty("jobs") ?: Runtime.runtime.availableProcessors()
 }
 
 task cleanReactNdkLib(type: Exec) {
+    ignoreExitValue true
```

Comment:
why ignore exitvalue ?

Reply:
Because running `gradle clean` before building executes ndkbuild which in turn throws errors for not being able to find the build folders. `clean` should always succeed.